src/components/SQFormTransferTool/SQFormTransferTool.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import LoadingSpinner from '../LoadingSpinner';
+import SQFormDialog from '../SQFormDialog';
+import SQFormTransferProductPanels from './SQFormTransferProductPanels';
+import {noop} from '../../utils';
+import type {FormikHelpers, FormikValues} from 'formik';
+import type {TransferProduct} from './types';
+
+export type SQFormTransferToolProps<Values extends FormikValues> = {
+  initialValues: Values; // This will be removed SC3-1810
+  /** boolean to indicate if the modal should show a loading indicator */
+  isLoading: boolean;
+  /** That data that drives the content of the modal and it's accordions */
+  transferProducts: Array<TransferProduct>;
+  /** The current open/closed state of the Dialog */
+  isOpen: boolean;
+  /** Callback function invoke when the user clicks the primary button */
+  onSave: (
+    values: Values,
+    formikHelpers: FormikHelpers<Values>
+  ) => void | Promise<unknown>;
+};
+
+export default function SQFormTransferTool<Values extends FormikValues>({
+  isLoading,
+  isOpen,
+  onSave,
+  transferProducts,
+  initialValues,
+}: SQFormTransferToolProps<Values>): React.ReactElement {
+  // TODO Validation: this will be internal and the consumer need not worry about it. It should be part of SC3-1810 processing each step
+  // TODO Initial Values: like validation this will be hidden from the consumer as they should not need it. SC3-1810 steps
+  // TODO Transfer: transfers will be handled via consumer supplied callback. SC3-1811
+
+  return (
+    <SQFormDialog
+      isOpen={isOpen}
+      onSave={onSave}
+      onClose={noop}
```

Comment:
is this noop intended to be permanent here?

Reply:
Probably....
Until something requires consuming the MUI 5 dialog directly I think I would prefer to use SQForm Dialog. Which requires an onClose Prop. The only use cases for exiting the dialog will require 'saving'. 

That being said, I am definitely using the button's weirdly here. My 'save' action is on the secondary to match the style of a comp. 

So as I get a bit further I am waiting to see if using SQFormDialog will really work for this or not. 
Since it may not, I am kinda playing fast and loose with things.

All that being said I have talked myself into adding a TODO comment here to re-evaluate after I get a bit further into the requirements.

Reply:
If we're using the buttons backwards, that seems like something we would want to take a look at. I think it's going to heavily depend on how it's consumed once it gets a little further along.

Reply:
Like above there are TODO's all over this given the current requirements. There is one on line 34. I agree this will depend on how its consumed, hence the todos. I am not a huge fan of littering the component with them, but I don't have a ton of choice yet.